tools/update-llhttp.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+set -e
```

Comment:
This is what I personally use.

```suggestion
#!/bin/bash
set -euo pipefail
shopt -s inherit_errexit
```

Then again, I don't know how widely supported this is beyond Linux.